release-notes.md

```diff
@@ -12,6 +23,15 @@ The following updates were made in this release:
 
 * Updated the use of the ACPPlaces `getNearbyPointsOfInterest` API to handle error scenarios more effectively.
 
+## July 22, 2019
+
+**Android Places 1.3.0**
+
+* Added a new API that clears out all Places-related data from shared state, in-app memory, and Shared Preference.
+* Fixed an issue where shared state was not getting updated during application start.
+* Fixed a bug where getNearbyPointsOfInterest callback was returning error code SERVER_RESPONSE_ERROR instead of CONNECTIVITY_ERROR on no network.
+* getNearbyPointsOfInterest API (without the errorCallback) will have the successCallback called with empty poi list, in case of error retrieving the nearby points of interest.
```

Comment:
please reword to read as follows: 

"If there is an error when retrieving nearby POIs, `getNearbyPointsOfInterest` API (without the errorCallback) will have the `successCallback` called with empty POI list.